packages/table/src/Table/TableWithVS.story.tsx

```diff
@@ -118,7 +120,7 @@ export const Basic: StoryFn<StoryTableProps> = args => {
         ref={tableContainerRef}
         className={virtualScrollingContainerHeight}
       >
-        <TableHead>
+        <TableHead isSticky>
```

Comment:
why the change to `isSticky`?

Reply:
Doesn't necessarily need to be on all stories, but IMO it's a better demo experience when the headers don't scroll out of view

Reply:
<img width="314" alt="Screenshot 2024-02-21 at 10 36 24 AM" src="https://github.com/mongodb/leafygreen-ui/assets/5740159/8eb9ed64-18b7-4281-883a-7d44d64cd2ed">

 I think this is causing z-index issues. The nested arrow is above the header. The same happens with checkboxes.

Reply:
Hmm. Opened this ticket https://jira.mongodb.org/browse/LG-4037

Reply:
Resolved 👍